CONTRIBUTING.md

```diff
@@ -6,9 +6,11 @@ and follow the [guidelines](https://jazzband.co/about/guidelines).
 
 ## Project Contribution Guidelines
 
-Here are a few additional or emphasized guidelines to follow when contributing to pip-tools:
+Here are a few additional or emphasized guidelines to follow when contributing to `pip-tools`:
 
-- Install pip-tools in development mode and its test dependencies with `pip install -e .[testing]`.
+- To set up your project's environment, have a look at [Python Packaging User Guide](https://packaging.python.org)
+  and [Real Python Tutorials](https://realpython.com)
+- Install pip-tools in development mode and its test dependencies with `pip install -e '.[testing]'`.
```

Comment:
Let's avoid telling people to directly access the extras. We don't have a linter that would enforce a person changing the name of the extra in `setup.cfg` or some setting in `tox.ini` to also make sure to change the guide and keep it in sync at all times. Why did you drop the `--devenv` thing anyway? It would create a venv with deps identical to what tox has.

Reply:
I now suggest to install tox and use the devenv option specified in tox's documentation. I am only suggesting one command line so we won't have to keep this doc up to date.